src/hooks/protections/receive_net_message.cpp

```diff
@@ -117,7 +117,12 @@ namespace big
 					player->is_spammer = true;
 					if (g.session.kick_chat_spammers)
 					{
-						dynamic_cast<player_command*>(command::get(RAGE_JOAAT("breakup")))->call(player, {});
+						if(g_player_service->get_self()->is_host())
+							dynamic_cast<player_command*>(command::get(RAGE_JOAAT("breakup")))->call(player, {}),
+							NETWORK::NETWORK_SESSION_KICK_PLAYER(player->id());
```

Comment:
Indeed, I put both in just in case. They are both within that if check.

Reply:
However if a player wants to use breakup kick for the cheating leave message it might be best to leave it in as an option.

Maybe we can make a host kick command with a switch case depending on what the preferred kick method is?